.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 
 env:
   ghc_version: '9.2.8'
-  hlint_version: '3.4.1'
+  hlint_version: '3.5'
```

Comment:
o/t: We might want to upgrade to version 3.8?  (and to a newer GHC) at some point...